File: python/BFS.py
graph = [[0, 1, 1, 1, 0, 0, 1], [1, 0, 1, 0, 0, 0, 0], [1, 1, 0, 0, 1, 0, 0], [1, 0, 0, 0, 0, 1, 0], [0, 0, 1, 0, 0, 1, 0], [0, 0, 0, 1, 1, 0, 1], [1, 0, 0, 0, 0, 1, 0]]

queue = [0]
visited = [0]

while len(queue) > 0:
    node = queue.pop()
    for key, value in enumerate(graph[node]):
        if value == 1 and key not in visited:
            visited.append(key)
            queue.insert(0, key)

for node in visited:
    print(node)

File: php/BFS.php
<?php

$graph = array(
      array(0,1,1,1,0,0,1),
      array(1,0,1,0,0,0,0),
      array(1,1,0,0,1,0,0),
      array(1,0,0,0,0,1,0),
      array(0,0,1,0,0,1,0),
      array(0,0,0,1,1,0,1),
      array(1,0,0,0,0,1,0),
   );

$queue = array(0);
$visited = array(0);

while(count($queue) > 0) {
  $node = array_pop($queue);
  foreach($graph[$node] as $key => $value){
      if($value == 1 && !in_array($key, $visited)) {
          array_push($visited, $key);
          array_unshift($queue, $key);
      }
  }
}

foreach($visited as $nodeVisit) {
  echo $nodeVisit."<br/>";
}

?>